robot-server/tests/runs/router/test_base_router.py

```diff
@@ -704,22 +720,29 @@ async def test_update_current_to_current_noop(
     result = await update_run(
         runId="run-id",
         request_body=RequestModel(data=run_update),
-        run_store=run_store,
-        run_view=run_view,
-        engine_store=engine_store,
+        run_store=mock_run_store,
+        engine_store=mock_engine_store,
     )
 
     assert result.content == SimpleBody(data=expected_response)
     assert result.status_code == 200
 
-    decoy.verify(run_store.upsert(run_resource), times=0)
-    decoy.verify(await engine_store.clear(), times=0)
+    decoy.verify(
+        mock_run_store.update_active_run(
+            run_id=run_resource.run_id,
+            is_current=run_update.current
+            if run_update.current is not None
+            else run_resource.is_current,
+        ),
+        times=0,
+    )
```

Comment:
I would simplify this to:

```suggestion
    decoy.verify(
        mock_run_store.update_active_run(
            run_id=matchers.Anything(),
            is_current=matchers.Anything()
        ),
        times=0,
    )
```

The intent of this test, according to the docstring, is that the router shouldn't do anything when a client tries to `PATCH {"current": true}` for a run that's already current. So if the router calls `run_store.update_active_run()` with *any* arguments, that's probably a mistake that this test should catch.